docs/source/development/index.rst

```diff
@@ -26,3 +26,51 @@ on the IPython GitHub wiki.
    pycompat
    config
    inputhook_app
+
+Making an IPython release
+=========================
```

Comment:
I actually meant just adding the new page to the TOC in this file, not putting the content in here, but it's fine.